airflow/contrib/hooks/gcp_sql_hook.py

```diff
@@ -296,7 +302,7 @@ def export_instance(self, instance, body, project_id=None):
         :return: None
         """
         try:
-            response = self.get_conn().instances().export(
+            response = self.get_conn().instances().export(  # pylint: disable=no-member
```

Comment:
We have quite a lot of the # pylint: disable=no-member - we can probably get rid of the pylint warning by annotating get_conn() with expected return type.

Reply:
Unfortunately this doesn't work 😕 In most cases `# pylint: disable=no-member` was used for discovery API. In those cases returned value is a `googleapiclient.discovery.Resource` object which has no public attributes and that's why pylint is complaining.

Reply:
Too bad :(